lib/srv/desktop/rdp/rdpclient/client_common.go

```diff
@@ -45,6 +45,10 @@ type Config struct {
 	// Encoder is an optional override for PNG encoding.
 	Encoder *png.Encoder
 
+	// AllowClipboard indicates whether the RDP connection should enable
+	// clipboard sharing.
+	AllowClipboard bool
```

Comment:
Is it worth flipping this around so the default is to allow clipboard sharing? Say, `DisallowClipboard bool`?

Reply:
The default (from Teleport RBAC's perspective) is still to allow clipboard sharing.

I don't think it really matters here, as we always explicitly provide this flag. Plus, I  find allow easier to reason about than the alternative.